services/thermal_mgr/thermal_mgr.c

```diff
@@ -42,19 +42,37 @@ void initThermalSystemManager(lm75bd_config_t *config) {
 }
 
 error_code_t thermalMgrSendEvent(thermal_mgr_event_t *event) {
-  /* Send an event to the thermal manager queue */
-
+  xQueueSend(thermalMgrQueueHandle, event, portMAX_DELAY);
   return ERR_CODE_SUCCESS;
 }
 
 void osHandlerLM75BD(void) {
-  /* Implement this function */
+  thermal_mgr_event_type_t event = THERMAL_MGR_EVENT_INTERUPT;
```

Comment:
this is of type `thermal_mgr_event_type_t`, but the receiving queue is expecting type `thermal_mgr_event_t`